dev/actors/actortypes/actor.md

```diff
@@ -28,7 +28,7 @@ Def.Actor{
 | Name | Type | Description |
 | :--- | :--- | ----------- |
 [Name]Command | function | The basis for any command. Check [Commands](#commands) for more information.
-Name | string | The name for the actor. Giving this a value allows this actor to be obtain by its name by any other actor. Check [Obtaining Childs and ActorFrame Levels](#obtaining-childs-and-actorframe-levels) for more information.
+Name | string | The name for the actor. Giving this a value allows this actor to be obtain by its name by any other actor. Check [Obtaining Children and ActorFrame Levels](#obtaining-children-and-actorframe-levels) for more information.
```

Comment:
Debatable. Might be referencing something in code, but it can probably be `Child` (singular)?